openassessment/xblock/student_training_mixin.py

```diff
@@ -49,8 +49,8 @@ def render_student_training(self, data, suffix=''):   # pylint:disable=W0613
 
         try:
             path, context = self.training_path_and_context()
-        except: # pylint:disable=W0702
-            msg = u"Could not render student training step for submission {}".format(self.submission_uuid)
+        except:  # pylint:disable=W0702
+            msg = u"Could not render Learner Training step for submission {}".format(self.submission_uuid)
```

Comment:
@dianakhuang I am not able to get to a state where I see these messages in the sandbox. I'm not clear how they render in the UI. Is it odd or OK that these strings do not end with a period or other punctuation at the end of the sentence?